Update publication deadline of LOOPS book 2

pamoroso opened the pull request:
Since @skaisler1 is still working on the book *Medley LOOPS Volume II: Tools and Utilities* this change updates the publication timeframe on the [Documentation page](https://interlisp.org/software/using-medley) from "in the late fall of 2024" to a more conservative "in the second half of 2025".

pamoroso committed:
Update publication deadline of LOOPS book 2

Update the publication timeframe of the book Medley LOOPS Volume II: Tools and Utilities from "in the late fall of 2024" to "in the second half of 2025" as the author is still working on it.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

pamoroso commented:
I'd appreciate your feedback.

masinter reviewed (approved):
this is OK and I will approve it, but I think it's unnecessary (so it's late?) and too specific -- why not just a (forthcoming)?

stumbo commented:
I'm like Larry's suggestion, let's just use **forthcoming** and not publish expected dates.  Paolo, would you accept that change?  If so, make it and I'm merge it.

pamoroso commented:
Yes, using "forthcoming" for volume 2 would be fine, I'll update the PR.

In my local fork of the repo I only submitted this change. How should I sync the repo? Discard 1 commit or update the branch?

![repo-sync](https://github.com/user-attachments/assets/c6712760-f83b-4369-85db-4baef27dbb31)

pamoroso committed:
Don't mention explicit publication deadline of LOOPS book 2

Change the publication deadline of LOOPS book 2 from an explicit timeframe to just "forthcoming".

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

pamoroso commented:
I went ahead and update the PR, I'll sync my fork once the PR is merged.
